src/grid.js

```diff
@@ -1,13 +1,35 @@
 import Split from 'split-grid'
-import { $ } from './utils/dom.js'
-
-Split({
-  columnGutters: [{
-    track: 1,
-    element: $('.vertical-gutter')
-  }],
-  rowGutters: [{
-    track: 1,
-    element: $('.horizontal-gutter')
-  }]
+import { $, $$ } from './utils/dom'
+
+const $editor = $('#editor')
+const $$layoutSelector = $$('.layout-preview')
+
+let splitInstance
+
+const formatGutters = gutter => ({
+  ...gutter,
+  element: $(gutter.element)
 })
+
+const setGridLayout = ({ gutters, style, type = '' }) => {
+  $editor.setAttribute('data-layout', type)
+  $editor.setAttribute('style', style)
+
+  $$layoutSelector.forEach(layoutEl => {
+    layoutEl.className = type === layoutEl.id ? 'layout-preview active' : 'layout-preview'
+  })
+
+  gutters = {
```

Comment:
I would create a new gutters variable with another name instead of replacing the one from the params.